mapper/sorting.go

```diff
@@ -0,0 +1,54 @@
+package mapper
+
+import (
+	"sort"
+	"strconv"
+
+	"github.com/ONSdigital/dp-api-clients-go/v2/population"
+)
+
+// sorts population categories by ID - numerically if possible, with negatives listed last
```

Comment:
comment/go-doc should start with the func name 
```suggestion
// sortCategoriesByID sorts population categories by ID - numerically if possible, with negatives listed last
```